scipy/stats/_stats_py.py

```diff
@@ -1999,7 +2185,8 @@ def jarque_bera(x, *, axis=None):
     ...     s = stats.skew(x, axis=axis)
     ...     k = stats.kurtosis(x, axis=axis)
     ...     return x.shape[axis]/6 * (s**2 + k**2/4)
-    >>> res = stats.monte_carlo_test(x, stats.norm.rvs, statistic)
+    >>> res = stats.monte_carlo_test(x, stats.norm.rvs, statistic,
+    ...                              alternative='greater')
```

Comment:
Oops! The JB statistic is "extreme" only when it is large, so it should have been `alternative='greater'` here.